json/midrise.json

```diff
@@ -1,5 +1,5 @@
 {
-  "schema": "https://github.com/rd2/tbd/blob/master/tbd.schema.json",
+  "schema": "https://github.com/rd2/tbd/blob/logger/tbd.schema.json",
```

Comment:
You might want to change these back to point at master right before merging

Reply:
Yup. I've had to do this in the past whenever I brought in changes to the schema. I did as you suggest - do a 2nd follow-up merge to _master_ to correct the links.